README.md

```diff
@@ -20,6 +20,23 @@ After you are done, run this in your shell to fetch the new dependency:
 $ mix deps.get
 ```
 
+## Configuration
+
+```
```

Comment:
Can you somehow mention what the default configuration is and where to put that file?

Reply:
Oh, that's for Phoenix, sorry... will update